SETUP/SETUP.BAT

```diff
@@ -518,6 +518,8 @@ vdelay 1000
 
 :PartitionManual
 vcls /g /fGray /bBlack
+echo NOTICE: FAT16 partition under 2Gb is needed for KERNL86.   

```

Comment:
After discussing the issue of FAT32 support in the 8086 kernel with the developers responsible for the kernel, they now plan on releasing 2 versions of the 8086 kernel for their next release. One with FAT32 and one with only FAT16. A notice may not be necessary in the near future. However, we can add a notice for the interim. But, we must update what you have provided to allow for multi-language support and add some color to make it stand out.

First, we need to make it support multiple languages. Change the echo  line to:

`vecho /g /t %LANGFILE% PARTITION_WARN`

The jump over to the [FD-NLS project resources for FDI-x86](https://github.com/shidel/fd-nls/blob/master/fdi-x86/SETUP.EN#L22) and add a line before or after **PARTITION_AUTO** that reads as follows:

`PARTITION_WARN=/fYellow NOTICE: /fGray FAT16 partition under 2Gb is needed for KERNL86.`  

This will allow additional languages to add translations for the warning.

Thank you.